xla/service/collective_pipeliner_test.cc

```diff
@@ -2228,5 +2229,90 @@ ENTRY main.3813_spmd {
   ASSERT_IS_OK(verifier.Run(module.get()).status());
 }
 
+TEST_F(CollectivePipelinerTest, PipelineBackwardIncludeInvariantInChain) {
```

Comment:
I assume this is the case that you care about. Can we also add a minimal test case with nothing but a collective + bcast constant?

Reply:
yup, i minimized this test a bit more.